tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -366,7 +374,29 @@ func (s *storeImpl) UpsertMany(ctx context.Context, objs []*{{.Type}}) error {
     } else if !ok {
         return sac.ErrResourceAccessDenied
     }
-    {{- end}}
+    {{- else if .Obj.IsDirectlyScoped -}}
+    {{ template "defineScopeChecker" "READ_WRITE" }}
```

Comment:
I like this appraoch but it require `isInScope` to be dfined and it will copy logic from scope checker (which is easy but has many corner cases). Maybe we can do it later to extract isInScope logic from scope checker (possibly to sac tree) so we can use it here.